backend/src/BuiltinExecution/Libs/UInt16.fs

```diff
@@ -0,0 +1,385 @@
+module BuiltinExecution.Libs.UInt16
+
+open FSharp.Control.Tasks
+open System.Threading.Tasks
+
+open System.Numerics
+
+open Prelude
+open LibExecution.RuntimeTypes
+open LibExecution.Builtin.Shortcuts
+
+module VT = ValueType
+module Dval = LibExecution.Dval
+
+let types : List<BuiltInType> = []
+let constants : List<BuiltInConstant> = []
+
+module ParseError =
+  type ParseError =
+    | BadFormat
+    | OutOfRange
+
+  let toDT (e : ParseError) : Dval =
+    let (caseName, fields) =
+      match e with
+      | BadFormat -> "BadFormat", []
+      | OutOfRange -> "OutOfRange", []
+
+    let typeName =
+      TypeName.fqPackage "Darklang" [ "Stdlib"; "UInt16" ] "ParseError" 0
+    DEnum(typeName, typeName, [], caseName, fields)
+
+
+let fn = fn [ "UInt16" ]
+
+let fns : List<BuiltInFn> =
+  [ { name = fn "mod" 0
```

Comment:
double-check the mod and remainder functions for uint8 and uint16. Since we're working with unsigned numbers, we can keep just one of them, as there are no negative values to make them behave differently